src/App.css

```diff
@@ -175,4 +175,10 @@ h3 {
 /* stylizes tab component here to avoid putting styles in editor*/
 .tab-content {
   margin-top: 1rem;
+}
+
+/* custom label style*/
```

Comment:
This comment is not terribly helpful. It would be more helpful to explain where this is used in our components.

Reply:
I assumed that we might apply the identical label style to another component. I have updated the comments specifying the component used.